clients/unshaded/src/main/java/tachyon/client/netty/NettyRemoteBlockReader.java

```diff
@@ -88,4 +93,18 @@ public ByteBuffer readRemoteBlock(String host, int port, long blockId, long offs
       throw new IOException(e);
     }
   }
+
+  /**
+   * Clear the previous read response, release the resource the response references.
+   *
+   * @return true if the response is cleared, or there is nothing needs to be cleared.
+   */
+  public boolean clearReadResponse() {
```

Comment:
I think it would be nice if the "RemoteBlockReader" interface included a close() method, so other classes just have to close() this reader when desired. Maybe "RemoteBlockReader" should extend "Closeable"?